index.html

```diff
@@ -49,10 +73,208 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
 
 
 
+
+    <div class="main">
+    
+        <div class="Works">
+          <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot1.png" alt=""></div>
+          
+          <div class="tonic">
+              <div><h1>Tonic</h1></div>
+              <div class="flex1">
+                  <div>Caponi</div>
+                  <div class="round"></div>
+                  <div>back End</div>
+                  <div class="round"></div>
+                  <div>2015</div>
+              </div>
+              <div>
+                  <p>
+                      A dayly selection of privatelypersonalized reads:no accounts <br>
+                      or sign-ups required.
+                  </p>
+              </div>
+              <div >
+                  <ul class="languages">
+                  <li class="language">HTML</li>
+                  <li class="language">CSS</li>
+                  <li class="language">javascript</li>
+                  </ul>
+              </div>
+              <div><button class="see-project" type="button">See project</button></div>
+          </div>
+            </div>
+          
+      
+      
+      
+            <div class="Works">
+              <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot2.png" alt=""></div>
+      
+      
+              <div class="tonic">
+                  <div><h1>Muti-Post <br>Stories</h1></div>
+                  <div class="flex1">
+                      <div>Caponi</div>
+                      <div class="round"></div>
+                      <div>back End</div>
+                      <div class="round"></div>
+                      <div>2015</div>
+                  </div>
+                  <div>
+                      <p>
+                          A dayly selection of privatelypersonalized reads:no accounts <br>
+                          or sign-ups required.
+                      </p>
+                  </div>
+                  <div >
+                      <ul class="languages">
+                      <li class="language">HTML</li>
+                      <li class="language">CSS</li>
+                      <li class="language">javascript</li>
+                      </ul>
+                  </div>
+                  <div><button class="see-project" type="button">See project</button></div>
+              </div>
+                </div>
+           
+          
+          
+          
+                <div class="Works">
+                  <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot3.png" alt=""></div>
+          
+          
+                  <div class="tonic">
+                      <div><h1>Tonic</h1></div>
+                      <div class="flex1">
+                          <div>Caponi</div>
+                          <div class="round"></div>
+                          <div>back End</div>
```

Comment:
- Kindly make sure your design sticks to the Figma design, by using the right border radius and background color.
- In my opinion, the border radius of the image must be the same as the card
Your design:
![Screenshot from 2023-02-21 15-33-24](https://user-images.githubusercontent.com/97834160/220359933-465647dc-ec55-4618-97b6-70b376024ee9.png)
Figma:
![Screenshot from 2023-02-21 15-38-14](https://user-images.githubusercontent.com/97834160/220360707-12be3c40-924f-49ce-939c-bfc358990431.png)